servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -548,6 +548,10 @@ private static ReferenceMetadata extractReferenceMetadata(ReferenceInfo<CommitMe
       builder.commitMetaOfHEAD(
           addHashToCommitMeta(refWithHash.getHash(), refWithHash.getHeadCommitMeta()));
     }
+    if (0L != refWithHash.getCommitSeq()) {
+      found = true;
+      builder.numTotalCommits(refWithHash.getCommitSeq());
```

Comment:
we should only add that if `fetchAdditionalInformation` was set to `true`, otherwise we'll be breaking the Python client (since it doesn't know anything about `ReferenceMetadata` yet.